enhancements/update/cvo-log-level-api.md

```diff
@@ -0,0 +1,327 @@
+for the CVO. The CVO will dynamically change the verbosity of its logs based on a
+value provided in the new resource. The CRD is described in more detail in the
+**Implementation Details** section. A new CRD is created to better differentiate 
+between the cluster version and the CVO configuration.
+
+Four log levels will be available. The lowest level being the
+current default log level being used by the Cluster Version Operator. The exact 
+log levels are defined as per the existing enum [`LogLevel`][LogLevelType] that 
+is used in the new `ClusterVersionOperator` CRD. See the **Implementation 
+Details** section for more details.
+
+[LogLevelType]: https://github.com/openshift/api/blob/f89ab92f1597eaed4de5b947c1781adde2bf42fb/operator/v1/types.go#L94-L110
+
+### Workflow Description
+
+Given a cluster administrator and a working cluster for which the administrator is responsible.
+
+**Cluster administrator** is a human user responsible for managing the cluster.
+
+1. The cluster administrator notices an issue in the cluster and chooses to
+   troubleshoot the issue.
+2. The cluster administrator, after some troubleshooting, notices that the logs
+   of the Cluster Version Operator (CVO) might help.
+3. The cluster administrator notices that the logs are not detailed enough to
+   troubleshoot the issue.
+4. The cluster administrator raises the log level from the default value to a
+   more verbose level by simply modifying the new `ClusterVersionOperator`
+   resource named `cluster` via the web console or by patching the resource by
+   using the CLI.
+5. The cluster administrator fixes the issue in the cluster.
+6. The cluster administrator notices that the CVO outputs too many logs for the
+   administrator's liking.
+7. The cluster administrator lowers the log level of the CVO to the lowest 
+   level, the default level.
+8. The cluster administrator is now a happy cluster administrator.
+
+### API Extensions
+
+The enhancement proposes to create a new `ClusterVersionOperator` CRD. A new
+`ClusterVersionOperator` resource will only impact the CVO logging level.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+A hosted CVO is located in the management cluster and accesses the hosted API
+server. As the new CR will be part of the OCP payload, it will be applied to the
+hosted cluster.
+
+It is needed to address the fact that the hosted cluster API server will now 
+provide access to the new CR that represents the CVO configuration. A 
+hosted cluster administrator could set the log level of the hosted CVO in the
+management cluster, thus affecting the storage space of the management cluster.
+
+In HyperShift, this enhancement proposes the management cluster to reconcile 
+the new resource. The HyperShift will maintain a fixed value, the default 
+value, of the hosted CVO log level and thus will overwrite any user changes.
+Similarly, as the HyperShift does for [log levels of other operators that 
```

Comment:
This would also need to be protected with out VAP https://github.com/openshift/hypershift/blob/32bd90748fee1d113ae0ac35e5b6ab7d08342885/control-plane-operator/hostedclusterconfigoperator/controllers/resources/kas/admissionpolicies.go#L108

Reply:
Awesome, thanks for the link. VAP is explicitly referenced in the enhancement now.